src/main/java/ru/rt/restream/reindexer/ReindexerConfiguration.java

```diff
@@ -103,6 +106,17 @@ public ReindexerConfiguration dataSourceFactory(DataSourceFactory dataSourceFact
         return this;
     }
 
+    /**
+     * Allows customizing a {@link FieldConverterRegistry}.
+     * 
+     * @param customizer the {@link FieldConverterRegistry} customizer.
+     * @return the {@link ReindexerConfiguration} for further customizations
+     */
+    public ReindexerConfiguration fieldConverterRegistry(Consumer<FieldConverterRegistry> customizer) {
```

Comment:
Not used in code, no examples of use in tests, looks like a reserve for the future. Is it really necessary now?